tests/signUp/signUp.spec.js

```diff
@@ -0,0 +1,286 @@
+        await signUpButton.click()
+    })
+    test ('Sign Up with VALID data', async ({page})=>{
+        const popup = page.locator('app-signup-modal')
+
+        const inputName = popup.locator('#signupName')
+        await expect(inputName).toBeVisible()
+        await expect(inputName).toBeEnabled()
+        await expect(inputName).toBeEditable()
+
+        const inputLastName = popup.locator('#signupLastName')
+        await expect(inputLastName).toBeVisible()
+        await expect(inputLastName).toBeEnabled()
+        await expect(inputLastName).toBeEditable()
+
+        const inputEmail = popup.locator('#signupEmail')
+        await expect(inputEmail).toBeVisible()
+        await expect(inputEmail).toBeEnabled()
+        await expect(inputEmail).toBeEditable()
+
+        const inputPassword = popup.locator('#signupPassword')
+        await expect(inputPassword).toBeVisible()
+        await expect(inputPassword).toBeEnabled()
+        await expect(inputPassword).toBeEditable()
+
+        const inputRepeatPassword = popup.locator('#signupRepeatPassword')
+        await expect(inputRepeatPassword).toBeVisible()
+        await expect(inputRepeatPassword).toBeEnabled()
+        await expect(inputRepeatPassword).toBeEditable()
+
+        const submitBtn = popup.locator('button', {hasText: 'Register'})
+        await expect(submitBtn).toBeVisible()
+        await expect(submitBtn).toBeDisabled()
+
+        await inputName.fill('Ivanna')
+        await inputLastName.fill('Dub')
+        await inputEmail.fill('aqa-testing.idub@gmail.com')
+        await inputPassword.fill('12345qwerrtyY!')
+        await inputRepeatPassword.fill('12345qwerrtyY!')
+        await submitBtn.click()
+    })
+    test.afterEach(async ({page})=>{
+        const settingsBtn = page.getByRole('link', { name: 'Settings' })
+        await settingsBtn.click()
+        const removeAccountBtn = page.locator('button', {hasText: 'Remove my account'})
+        await removeAccountBtn.click()
+        const popupRemoveAccount = page.locator('app-remove-account-modal')
+        const confirmRemovingBtn = popupRemoveAccount.locator('.btn-danger')
+        await confirmRemovingBtn.click()
+    })
+})
+
+test.describe("Check validation for fields", ()=>{
+    test.beforeEach(async ({page})=>{
+        await page.goto("")
+        const signUpButton = page.locator('button', {hasText: 'Sign Up'})
+        await signUpButton.click()
+    })
+    test ('Sign Up with INVALID data for Name input', async ({page})=>{
+        const popup = page.locator('app-signup-modal')
+        const inputName = popup.locator('#signupName')
+        const submitBtn = popup.locator('button', {hasText: 'Register'})
+
+        // Negative: check required Name
+        await inputName.click()
+        await inputName.blur()
+        const errorMessageRequiredName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageRequiredName).toContain("Name required")
```

Comment:
use https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text